.travis.yml

```diff
@@ -13,16 +13,26 @@ jobs:
   include:
   - stage: test
     jdk: openjdk8
-    script: ./gradlew check :test
+    script:
+      - ./gradlew check :test
+      - bundle install
+      - bundle exec bugsnag-maze-runner
   - stage: test
     jdk: oraclejdk9
-    script: ./gradlew check :test
+    script:
+      - ./gradlew check :test
+      - bundle install
+      - bundle exec bugsnag-maze-runner
   - stage: test
     jdk: oraclejdk8
-    script: ./gradlew check :test
+    script:
+      - ./gradlew check :test
+      - bundle install
+      - bundle exec bugsnag-maze-runner
   - stage: test
     jdk: openjdk7
-    script: ./gradlew :test
+    script:
+      - ./gradlew check :test
```

Comment:
this stage appears to be missing the mazerunner step

Reply:
I removed this because it failed here https://travis-ci.org/bugsnag/bugsnag-java/builds/422924616?utm_source=github_status&utm_medium=notification I'll try adding it again